core/storage/email/gae_models.py

```diff
@@ -252,7 +270,7 @@ class BulkEmailModel(base_models.BaseModel):
     recipient_ids = ndb.JsonProperty(default=[], compressed=True)
     # The user ID of the email sender. For site-generated emails this is equal
     # to SYSTEM_COMMITTER_ID.
-    sender_id = ndb.StringProperty(required=True)
+    sender_id = ndb.StringProperty(required=True, indexed=True)
```

Comment:
will it work fine for the existing models? (Existing models will not be indexed, right?)

Reply:
No, removed.